src/apply-configuration.coffee

```diff
@@ -13,8 +14,23 @@ coerceToArray = (value) ->
   else
     return value
 
-applyConfiguration = (config) ->
 
+applyLoggingOptions = (options) ->
+  # coerce color to bool
+  if options.color == 'false'
+    options.color = false
+  else if options.color == 'true'
```

Comment:
Only strings have special handling here. I would change comment in a way to emphasise it (e.g. `coerce string color to bool, leave others`).